src/content/en/2021/css.md

```diff
@@ -232,7 +232,7 @@ As for what `!important` gets applied to, as with last year, it’s `display`, w
 
 ### Selector specificity
 
-Many CSS methodologies recommend that authors restrict themselves to single classes in order to squash all selector specificity into a single layer that is more easily managed.  The [BEM methodology](​​https://en.bem.info/methodology/css/), for example, was found on 34% of all pages.  The 10th percentile of median selector specificity shows further evidence of this type of thinking, where both desktop and mobile specificity averages at (0,1,0).  This is in line with last year’s findings, as are nearly all the medians — with the exception of mobile’s 25th percentile, which rose a little bit.
+Many CSS methodologies recommend that authors restrict themselves to single classes in order to squash all selectors specificity into a single layer that is more easily managed.  The [BEM methodology](https://en.bem.info/methodology/css/), for example, was found on 34% of all pages.  The 10th percentile of median selector specificity shows further evidence of this type of thinking, where both desktop and mobile specificity averages at (0,1,0).  This is in line with last year’s findings, as are nearly all the medians — with the exception of mobile’s 25th percentile, which rose a little bit.
```

Comment:
I can see this as either "selector specificity" or "selectors' specificity"

Reply:
I defer this to you, either works. I don’t know that term very well

Reply:
Let’s change it to “selectors' specificity”.